src/app/functions/server/onerep.ts

```diff
@@ -439,3 +449,47 @@ export async function fetchAllPages<Data>(
 
   return dataList.flat();
 }
+
+// Local instance map to cache results to prevent excessive API requests
+// Would be nice to share this cache with other pod via Redis in the future
+const profileStatsCache = new Map<string, ProfileStats>();
```

Comment:
Hm this would work... I was thinking of having a cron job that did this and populated the database, which we could query from Grafana as well for graphing and alerts. wdyt?

Reply:
What would that table look like? Since telegraf does the time series magic automatically, it would just be one entry in the db?

Reply:
kind of like subscriber count is just a count(*) that telegraf charts overtime by itself

Reply:
I put a simple example in https://github.com/mozilla/blurts-server/pull/3779 - I think the query would need to be a slightly more complicated but not much more (`select count(*) where current >= max` or so)

Reply:
This makes sense, I'll wait till #3779 is merged then